tests/alive-tv/attrs/nocapture.src.ll

```diff
@@ -0,0 +1,32 @@
+@x = global i8* null
+
+define void @f1(i8* nocapture %p) {
+  store i8* %p, i8** @x
+  ret void
+}
+
+define void @f2(i8* nocapture %p0) {
+  %p = getelementptr i8, i8* %p0, i32 1
+  store i8* %p, i8** @x
+  ret void
+}
+
+define i8* @f3(i8* nocapture %p) {
+  ret i8* %p
+}
+
+define i8* @f4(i8* nocapture %p) {
+  %p2 = getelementptr i8, i8* %p, i32 1
+  ret i8* %p2
+}
+
+;define {i8*, i8*} @f5(i8* nocapture %p) {
```

Comment:
why are these commented out?

Reply:
Because insertvalue is not supported yet.
```
ERROR: Unsupported instruction:   %v = insertvalue { i8*, i8* } undef, i8* %p, 1
```
These tests are needed for checking whether aggregate value return is working correctly. Should be uncommented after insertvalue support is added

Reply:
what if the return value is a vector instead?  Then you can use insertelement

Reply:
Yes, it works. Updated the test